SU-HSDO/suhumsci
# feat(STN-196): masthead dark mode variant color

# READY FOR REVIEW

## Summary
This work adds support for the masthead dark variant, inverting the main menu text colors and icons.  This is applied to the main menu as well as the mobile menu.  The desktop main menu dropdown (level 2) should remain in the light mode as it is in the default mode.

Jira Card: [STN-196](https://sparkbox.atlassian.net/browse/STN-196)

## Need Review By (Date)
10/16

## Urgency
low

## Steps to Test
1. run `npm run test`
2. Go to the theme settings to turn on the dark masthead variant, (appearance > settings > [theme colorful or traditional] > local masthead setting)
3. The masthead should change to the dark variant color pairings
4. Verify the text is inverted on the main menu and all icons work in both mobile size and desktop. Note: the mobile menu will be a full dark menu variant to match the masthead color.
5. After this is verified, swap to the other theme: traditional or colorful and verify the same steps.
6. You can also verify the different color pairings to confirm they work as they should as well

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
